### PR TITLE
Fix: update passcode creation screen for design review

### PR DESCRIPTION
**Pull request by billypchan**
## What's new in this PR?

Update UI styling.

### CODE REVIEW

**Comment by johnxnguyen** [comment]
Should the base branch be `develop`?

**Comment by billypchan** [comment]
fixed target branch

File: Wire-iOS/Resources/Base.lproj/Localizable.strings [hunk 1]
```diff
@@ -1780,24 +1780,25 @@
...
 
 // Custom app lock - create passcode validation
-"create_passcode.validation.too_short" = "8 Characters long";
-"create_passcode.validation.no_lowercase_char" = "1 Lowercase letter";
-"create_passcode.validation.no_uppercase_char" = "1 Capital letter";
-"create_passcode.validation.no_special_char" = "1 Special character";
-"create_passcode.validation.no_number" = "1 Number";
+"create_passcode.validation.too_short" = "At least 8 characters long";
+"create_passcode.validation.no_lowercase_char" = "A lowercase letter";
+"create_passcode.validation.no_uppercase_char" = "A uppercase letter";
```
Comment (johnxnguyen) [inline Wire-iOS/Resources/Base.lproj/Localizable.strings:1795]:
```suggestion
"create_passcode.validation.no_uppercase_char" = "An uppercase letter";
```

File: Wire-iOS/Sources/Helpers/UILabel+Convenience.swift [hunk 2]
```diff
@@ -51,4 +51,23 @@ extension UILabel {
 
         return label
     }
+    
+    // MARK: - passcode
+    
+    static func createHintLabel(variant: ColorSchemeVariant) -> UILabel {
+        let label = UILabel()
+        
+        label.font = UIFont.smallRegularFont.withSize(10) ///TODO: dynamic?
```
Comment (johnxnguyen) [inline Wire-iOS/Sources/Helpers/UILabel+Convenience.swift:60]: TODO

File: Wire-iOS/Sources/UserInterface/Overlay/CustomAppLock/Setup/PasscodeSetupViewController.swift [hunk 3]
```diff
@@ -67,8 +67,18 @@ final class PasscodeSetupViewController: UIViewController {
...
+        paragraphStyle.maximumLineHeight = 20
+        
+        let headingText =  NSAttributedString(string: "create_passcode.info_label".localized, attributes: [
+            .paragraphStyle: paragraphStyle
+        ]) && UIFont.normalRegularFont && textColor
+        let highlightText = NSAttributedString(string: "create_passcode.info_label.highlighted".localized, attributes: [
+            .paragraphStyle: paragraphStyle
+        ]) && FontSpec(.normal, .bold).font!  && textColor
+        
+        //TODO: line hgh
```
Comment (johnxnguyen) [inline Wire-iOS/Sources/UserInterface/Overlay/CustomAppLock/Setup/PasscodeSetupViewController.swift:81]: TODO

File: Wire-iOS/Sources/UserInterface/Overlay/CustomAppLock/Setup/PasscodeSetupViewController.swift [hunk 4]
```diff
@@ -67,8 +67,18 @@ final class PasscodeSetupViewController: UIViewController {
...
+        let paragraphStyle = NSMutableParagraphStyle()
+        paragraphStyle.minimumLineHeight = 20
+        paragraphStyle.maximumLineHeight = 20
+        
+        let headingText =  NSAttributedString(string: "create_passcode.info_label".localized, attributes: [
+            .paragraphStyle: paragraphStyle
+        ]) && UIFont.normalRegularFont && textColor
+        let highlightText = NSAttributedString(string: "create_passcode.info_label.highlighted".localized, attributes: [
+            .paragraphStyle: paragraphStyle
+        ]) && FontSpec(.normal, .bold).font!  && textColor
```
Comment (johnxnguyen) [inline Wire-iOS/Sources/UserInterface/Overlay/CustomAppLock/Setup/PasscodeSetupViewController.swift:79]: The readability could be improved by storing using a base attributes constant (paragraph style and text color).

**Comment by johnxnguyen** [comment]
I think there will be more adjustments to copy.

**Comment by billypchan** [comment]
@johnxnguyen for other copy update, I will change them in other PRs of corresponding screens.

**Comment by johnxnguyen** [comment]
There should be at least one for this screen. "Create a passcode" instead of "Create a Passcode". Check with design (I left the comment on Zeplin but maybe they haven't seen it yet).

**Pull request merged**